src/layouts/index.scss

```diff
@@ -15,3 +15,21 @@ body {
 @import 'theme.scss';
 
 @include react-md-everything;
+
+@import '~carbon-components/scss/globals/grid/_grid.scss';
```

Comment:
- [ ] It took some digging around, but it looks like there is a way to import all of the styles:
```scss
@import '~carbon-components/scss/globals/scss/styles';
```

Reply:
- [ ] Sorry, I meant to say that you can remove all of the `@import`s of Carbon Components and replace them all with the single `@import`:
```scss
@import '~carbon-components/scss/globals/scss/styles';
```